info/diktat-kotlin-coding-style-guide-en.md

```diff
@@ -941,46 +949,57 @@ This international code style prohibits non-latin (non ASCII) symbols in the cod
 
 #### <a name="r3.6"></a>Rule 3.6 line break style rules if the line is split
 
-  1. Compared to Java Kotlin allows not to to put semicolumn (';') after each statement separated by newline.
-     There should be no redundant semicolumn at the end of lines.
-     
-     In case when newline is needed to split the line, it should be placed after operators like &&/||/+/e.t.c
-     But newline should be placed before operators like ('.', '?.', '?:', '::', e.t.c), operator !! should not be separated from the value it is checking.
-     
-     Note, that you need to follow functional style:
-     ```kotlin
-        val value = otherValue!!
-                .map(x -> x)
-                .filter(true)
-                .size      
-     ```
-      
-  2. Newline should be placed after assignment operator ('=')
-  3. The name of a function or constructor should not be split by a newline from the opening brace '('. Brace follows immediately after the name without any spaces.   
-  4. Newline should be placed right after the comma (',')
-  5. If lambda expression has one-argument, a newline can appear after an arrow, if an expression has more than one argument before an arrow - breakpoint should appear after the brace.
+1. Compared to Java Kotlin allows not to to put semicolumn (';') after each statement separated by newline.
+ There should be no redundant semicolumn at the end of lines.
+ 
+ In case when newline is needed to split the line, it should be placed after operators like &&/||/+/e.t.c
+ But newline should be placed before operators like ('.', '?.', '?:', '::', e.t.c), operator !! should not be separated from the value it is checking.
+ 
+ Note, that you need to follow functional style:
+ ```kotlin
+    val value = otherValue!!
+            .map(x -> x)
+            .filter(true)
+            .size      
+ ```
   
-  Recommended examples：
+2. Newline should be placed after assignment operator ('=')
+3. The name of a function or constructor should not be split by a newline from the opening brace '('. Brace follows immediately after the name without any spaces.   
+4. Newline should be placed right after the comma (',')
+5. In lambda statements and `when` conditions, if it's body contains more than one line, the newline should be placed after an arrow. See examples below:
 
-     ```kotlin
-     val lambda =
-         (label: String, value: Long) -> {
-             // ...
-         }
-     
-     val predicate = str ->
-         longExpressionInvolving(str)
-     ```
+Bad example:
+```kotlin
+    value.map { name -> foo()
+        bar()
+    }
+```
+
+Recommended examples：
+```kotlin
+when (node.elementType) {
+         SOMETHING -> 
+             foo()
+             bar()
```

Comment:
I don't think multiple statements can be not wrapped in braces - at least I couldn't make similar compilable example.

Reply:
yes, true, will fix it now